### DISCUSSION TITLE
Why is the dependency so big all of the sudden?

### DISCUSSION THREAD

**Discussion by Selbi182** [xerial/sqlite-jdbc#873] [Q&A]
I've recently noticed that I haven't updated this dependency since 3.34.0. So I did that, currently at 3.40.1.0 (I know there are even newer versions out now, but that's not of any concern right now). From a technical standpoint, everything is fine and still works as intended.

However, I've noticed that since the update, my compiled Spring "bootJar" has blown up in size. Looking into the compiled Jar and sorting dependencies by size shows that this JDBC driver alone **takes up almost 13 MB**, making it four times as large as even Tomcat Core:

![grafik](https://user-images.githubusercontent.com/8850085/228837836-4d2ca0f8-a1d4-4662-b349-20026375b20a.png)

Why is it so big? And more importantly, what happened between 3.34 and now that almost doubled the size of the dependency? [It used to be 7 MB](https://mvnrepository.com/artifact/org.xerial/sqlite-jdbc/3.34.0), which isn't exactly small either, but I don't understand what happened to make it so big.

**Comment by gotson** [comment]
The library ships with native binaries for multiple platforms. We added many platforms in the past year or so.

**Comment by Selbi182** [comment]
I see. Understandable. Well, guess I just gotta accept that fact. Thanks for responding 🙂
